backend.py

```diff
@@ -84,6 +95,8 @@ def index():
     frame = int(request.args.get('frame'))
     if frame >= len(os.listdir(FRAME_DIR)):
         return {'result': None}
+    if frame == -1:
+        return json.dumps({'width': width, 'height': height, 'total_frames': len(os.listdir(FRAME_DIR))})
```

Comment:
Instead of setting it to return this at frame=-1, it's probably better practice to make a new route for this like
```python
@app.route('/size')
def get_size():
     ...
     return json.dumps({'width': width, 'height': height, 'total_frames': len(os.listdir(FRAME_DIR))})
```

And then call it in the front-end like
```js
xhr.open('GET', 'http://127.0.0.1:5000/size');
```

Reply:
Aha, I thought it would be something like that but wasnt sure and didnt want to spend time finding out, I will fix that now